cluster-autoscaler/cloudprovider/aws/aws_util.go

```diff
@@ -31,9 +31,10 @@ import (
 )
 
 var (
-	ec2MetaDataServiceUrl        = "http://169.254.169.254/latest/dynamic/instance-identity/document"
-	ec2PricingServiceUrlTemplate = "https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/%s/index.json"
-	staticListLastUpdateTime     = "2019-10-14"
+	ec2MetaDataServiceUrl          = "http://169.254.169.254/latest/dynamic/instance-identity/document"
+	ec2PricingServiceUrlTemplate   = "https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/%s/index.json"
+	ec2PricingServiceUrlTemplateCN = "https://pricing.cn-north-1.amazonaws.com.cn/offers/v1.0/cn/AmazonEC2/current/%s/index.json"
```

Comment:
Minor: There're multiple partitions like AWS, AWS-cn, AWS-gov.  It would be great to have all of them